memcached-spring-boot-autoconfigure/build.gradle

```diff
@@ -15,7 +15,9 @@ dependencies {
     optionalImplementation 'org.springframework.boot:spring-boot-actuator'
     optionalImplementation 'org.springframework.cloud:spring-cloud-context'
     optionalImplementation 'io.micrometer:micrometer-core'
+    implementation 'org.springframework:spring-context-support'
```

Comment:
Could you also group `implementation` dependencies together

Reply:
Sure